community/kernel/src/test/java/org/neo4j/kernel/GraphDatabaseFacadeFactoryTest.java

```diff
@@ -143,7 +146,8 @@ public LifeSupport createLife()
             @Override
             protected EditionModule createEdition( PlatformModule platformModule )
             {
-                return null;
+                EditionModule edition = Mockito.mock( EditionModule.class, Answers.RETURNS_DEEP_STUBS.get() );
```

Comment:
Is there a reason for that? I thought both are basically equivalent and public API